### ISSUE TITLE
SC 1.3.3: Sensory Characteristics

### ISSUE THREAD

**Issue by erikkroes** [w3c/wcag#1534]
The criterion states: "Instructions provided for understanding and operating content do not rely solely on sensory characteristics of components such as shape, color, size, visual location, orientation, or sound. " In discussion today, we tripped a bit over "do not rely solely on sensory characteristics".

I (and others) read this as, it should not just rely on sensory characteristics as mentioned but should also rely on (at least) another characteristic. This clashes with the first example. 

**Example 1: A schedule of competitive events uses color and shape to distinguish the time of each event**
A table presents a list of times across the top row and a list of events in the first vertical column. The cell corresponding to the time of a particular event has a specific background color and diamond shaped glyph so it can be identified by color and shape.

In my interpretation, this example would be a failure as it relies on sensory characteristics, namely, shape and color.

Another interpretation was that it could rely on sensory characteristics, but not -a- (single) sensory characteristic. So if it depends on multiple, it's allowed. This aligns with Guideline 1.3: "Create content that can be presented in different ways (for example simpler layout) without losing information or structure. "
When you depend on multiple sensory characteristics, it's presented in different ways. (But it could lead to the exclusion of people with multiple disabilities)

How is this SC supposed to be read?

**Comment by bruce-usab** [comment]
Thanks for raising this issue @erikkroes. 

FWIW, I have always understood this SC to be about forbidding a _**single**_ sensory characteristic.  My recollection is that the first example was provided to make this clear.  I doubt the working group would want to modify the SC text, but certainly Understanding could be a little bit more explicit.

**Comment by patrickhlauke** [comment]
xref https://github.com/w3c/wcag/issues/1532 and this PR https://github.com/w3c/wcag/pull/767

**Comment by patrickhlauke** [comment]
in general, the idea (to me at least) for the SC is that you shouldn't refer to things (specific controls or content) purely by their visual  or audio characteristics (one or more) because somebody may not be able to see, hear, see and hear, etc. other SCs like 1.1.1,, 1.3.1, etc. guarantee that there are programmatic and text alternatives for "the thing", this SC guarantees that when there are *instructions* that talk about "the thing" that they don't just refer to the it by one visual/auditory characteristics

**Comment by mraccess77** [comment]
I agree with @patrickhlauke instructions would need to be available in a way that doesn't require perception of these things - this would generally mean describing it in a way that does not rely on any sensory characteristics.

**Comment by awkawk** [comment]
I agree that it would be good to finish off #767 as it seems to address this question.

**Comment by jake-abma** [comment]
(@erikkroes would be better next time to use the same issue we discussed, keeps it a bit more organized, thx and cheers!)

**Comment by alastc** [comment]
@erikkroes & @jake-abma, just to be clear is this a duplicate of #1532?

**Comment by erikkroes** [comment]
It is not.
